metadata/hostmetadata/host.go

```diff
@@ -83,13 +96,22 @@ type OS struct {
 func (o *OS) ToStringMap() map[string]string {
 	return map[string]string{
 		"host_kernel_name":    o.HostKernelName,
+		"host_kernel_release": o.HostKernelRelease,
 		"host_kernel_version": o.HostKernelVersion,
 		"host_os_name":        o.HostOSName,
-		"host_os_version":     o.HostOSVersion,
 		"host_linux_version":  o.HostLinuxVersion,
 	}
 }
 
+// Int8ArrayToByteArray converts an []int8 to []byte
+func Int8ArrayToByteArray(in []int8) []byte {
```

Comment:
It seems strange to export this when it doesn't have anything to do with host metadata directly.